.github/workflows/hash-backend-cd.yml

```diff
@@ -45,9 +42,27 @@ jobs:
   build-graph:
     name: Build and push HASH graph image
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
We should probably add `permissions` to all actions at some point.

Reply:
Yes we should. These are necessary specifically for the id-token vault auth though